fighthealthinsurance/common_view_logic.py

```diff
@@ -1209,3 +1210,15 @@ async def format_response(response: dict[str, str]) -> str:
             subbed_appeals_json
         )
         return interleaved
+
+def validate_semi_sekret(semi_sekret):
+    """
+    Validate that semi_sekret follows the expected format to prevent injection attacks.
+    """
+    if not semi_sekret or not isinstance(semi_sekret, str):
+        return False
+    if len(semi_sekret) != 36:
+        return False
+    if not re.match(r'^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$', semi_sekret):
+        return False
+    return True
```

Comment:
This should probably go in the form & also would be good to have a test to make sure that the semisekret we generate passes this